lesson 1/README.md

```diff
@@ -0,0 +1,363 @@
+
+### Hello World
+
+Copy paste into `HelloWorld.java` and read over the following code:
+
+```java
+public class HelloWorld {
+
+    public static void main(String[] args) {
+        System.out.println("Hello World!");
+    }
+
+}
+```
+
+A window button should appear next to the main method header. Click it to run your program!
+
+```
+Hello World!
+```
+
+Don't worry too much yet about what all the words in `public static void main(String[] args)` mean. For now, just remember that this is the **main method**, and that every java program must have a main method defined. This main method is what the Java Virtual Machine runs initially, and all subsequent code eventually stems from the main method.
+
+### `Account` and `MoneySaver`
+
+#### Classes
+
+Create two new classes, and copy paste the following boilerplates into them:
+
+```
+public class MoneySaver {
+
+    private long myMoney;
+    private Account myAccount;
+    private String name;
+
+
+    public MoneySaver(String name, long money) {
+        this.name = name;
+        myMoney = money;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Account getMyAccount() {
+        return myAccount;
+    }
+
+    public String toString() {
+        return name + ", " + "My balance is: "+ myMoney;
+    }
+
+    public void deposit(int amount) {
+    }
+
+    public void withdraw(int amount) {
+    }
+    
+    public void signUpForChecking(int amount) {
+    }
+
+    public static void main(String[] args) {
+    }
+}
+
+```
+
+```
+public class Account {
+
+    private MoneySaver owner;
+    private long amount;
+
+    public String toString() {
+        return "Account owner: " + owner.getName() + ", Account Balance: $" + amount;
+    }
+
+    public static Account largerAccount(Account acc1, Account acc2) {
+    }
+
+}
+
+```
+
+##### Constructors
+
+The constructor is a special method used to initialize instances of your class (just like Python's `__init__` method). Classes by default inherit a no-argument constructor from the default `Object` class. That means that even without epxlicitly defining a constructor, we can already initalize an `Account`:
+
+```Account a = new Account() // This works!```
+
+Often, you will want to initialize your object with some values. In this case you must define your own constructor.
+
+**Go ahead and implement the `Account` constructor. It should have input arguments `long amount, MoneySaver owner`**
+
+#### Fields
+
+You'll notice both classes also have fields. Fields are either attributes of members of a class (an `amount` describes an `Account`) or things that belong to the class (an `Account` has an `owner`). **Fields should almost exclusively be `private` unless you have a *really* good reason to make it `public`.** If you want an outside class to be able to access a field, you must create public getter/setter methods.
```

Comment:
really? I didn't learn about protected for a while when first learning Java....You can get away with only doing private and public when you start out.